client/my-sites/upgrades/domain-management/edit-contact-info/form-card.jsx

```diff
@@ -118,6 +120,26 @@ class EditContactInfoFormCard extends React.Component {
 		}
 	}
 
+	hasEmailChanged() {
+		return this.props.contactInformation.email === formState.getFieldValue( this.state.form, 'email' );
+	}
```

Comment:
`hasEmailChanged` just sounds more proper to me - I prefer to write code so that it reads as much as a proper sentence as possible. `isEmailSame` feels awkward to me.
And, actually, this function does check if the form's state has changed from its original value.